core/src/Baked/Theme/Default/DefaultThemeFeature.cs

```diff
@@ -108,12 +107,12 @@ public virtual void Configure(LayerConfigurator configurator)
 
                     dtc.Title = data.Label is not null ? l(data.Label) : null;
                     dtc.Exportable = true;
+                    dtc.Component.Data = Context.Parent(options: o =>
+                    {
+                        o.Prop = $"row.{c.Property.Name.Camelize()}";
+                    });
```

Comment:
when it is only one line, it is simpler to remove braces and use expression body syntax